source/_integrations/transmission.markdown

```diff
@@ -14,27 +14,15 @@ ha_codeowners:
 ha_domain: transmission
 ---
 
-The `transmission` integration allows you to monitor your downloads with [Transmission](https://www.transmissionbt.com/) from within Home Assistant and setup automation based on the information.
-
-## Setup
```

Comment:
This heading should not be removed. The integration is pretty old and the chances are high links point here.

Reply:
OK, restored the setup heading and moved the text back to that section. Should be good now I think.